test/e2e/e2e_test.go

```diff
@@ -107,12 +106,13 @@ func TestMySqlToKafka(t *testing.T) {
 	}()
 
 	// run mysql_kafka.yml file
-	cfg, err := config.Load()
+	cfg, err := config.Load("mysql_kafka.yml")
 	if err != nil {
 		t.Error(err)
 	}
-	command := cmd.New(utils.Logger, nil, cfg)
-	command.SetArgs([]string{"run", "mysql_kafka.yml"})
+	command := cmd.RunCmd(utils.Logger, nil, cfg)
+
+	command.SetArgs([]string{"mysql_kafka.yml"})
```

Comment:
I am not sure on the reason for these changes here. Was this broken earlier?? @scortier @StewartJingga

Reply:
🤔 not sure on the changes? was it broken before @scortier?

Reply:
seems like it is being used now

Reply:
@StewartJingga @GrayFlash Yes, the e2e test was breaking before. I think some cli changes were made before but it was not refactored here